.gn

```diff
@@ -1,4 +1,5 @@
 buildconfig = "//Meta/gn/build/BUILDCONFIG.gn"
 exec_script_whitelist = []
+export_compile_commands = [ ":*" ]
```

Comment:
This might confuse ides once the serenity build is unbroken and filled out, with duplicate rules per file with different compilers. But, for now it's fine 🙂

Reply:
Do we know what the Build folder structure will look like? With CMake, we have `Build/lagom` and (e.g.) `Build/x86_64`, each with their own compile_commands.json. So I just swap which one my .clangd file points at depending on what I'm working on. If that all changes with GN, yeah we'll have to revisit this